packages/enzyme-test-suite/test/RSTTraversal-spec.jsx

```diff
@@ -316,6 +316,115 @@ describe('RSTTraversal', () => {
       expect(nodes).to.deep.equal([node, divA, divB, divC, divD]);
     });
 
+    describe('support for arbitrary iterable children', () => {
+      const makeDivIterator = (lowerBound, upperBound) => {
+        let counter = lowerBound;
```

Comment:
`let counter = lowerBound + 'a'.charCodeAt(0);` would avoid needing to look up "a"'s charCode every time